app/models/pod.js

```diff
@@ -32,7 +33,7 @@ var Pod = Resource.extend(DisplayImage, {
     shell() {
       get(this, 'modalService').toggleModal('modal-shell', {
         model: this,
-        escToClose: false,
+        escToClose: true,
```

Comment:
Hitting escape inside the terminal should go to the terminal, for things like... anything in vim...